lib/parse.js

```diff
@@ -50,6 +50,9 @@ Parse.prototype._readRecord = function () {
     if (signature === 0x04034b50) {
       return self._readFile();
     }
+    else if (signature === 0x8074b50) {
+      return data.length;
+    }
```

Comment:
I'm not super familiar with the internals of how this works. Why does returning the length of the data work for this?

Reply:
Basically, it moves the us past this chunk in the stream.